.changeset/sweet-maps-rule.md

```
@@ -0,0 +1,5 @@
+---
+"task-master-ai": minor
+---
+
+Upgrade grok-cli ai provider to ai sdk v5
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)